### PR TITLE
feat: brk make organization_id computed

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: docs/data-sources/registry_image.md [hunk 1]
```diff
@@ -47,3 +45,4 @@ In addition to all above arguments, the following attributes are exported:
 - `size` - The size of the registry image.
 - `visibility` - The privacy policy of the registry image.
 - `tags` - The tags associated with the registry image
+- `organization_id` - The ID of the organization the image is associated with.
```
Comment (kindermoumoute) [inline docs/data-sources/registry_image.md:48]:
```suggestion
- `organization_id` - The organization ID the image is associated with.
```

File: docs/index.md [hunk 2]
```diff
@@ -103,11 +103,11 @@ resource "scaleway_instance_server" "web" {
...
 
-The Scaleway authentication is based on an **access key** and a **secret key**.
+The Scaleway authentication is based on an **access key**, and a **secret key**.
 Since secret keys are only revealed one time (when it is first created) you might
-need to create a new one in the section "API Tokens" of the
-[Scaleway console](https://console.scaleway.com/account/credentials).
-Click on the "Generate new token" button to create them. Giving it a friendly-name is recommended.
+need to create a new one in the section "API Tokens" of the [Scaleway console](https://console.scaleway.com/account/credentials).
+Click on the "Generate new token" button to create them.
+Giving it a friendly-name is recommended.
```
Comment (kindermoumoute) [inline docs/index.md:110]: We might have more links to fix here

```suggestion
need to create a new one in the section "API Keys" of the [Scaleway console](https://console.scaleway.com/project/credentials).
Click on the "Generate new API key" button to create them.
Giving it a friendly-name is recommended.
```

File: docs/resources/account_ssh_key.md [hunk 3]
```diff
@@ -23,14 +23,14 @@ The following arguments are supported:
...
 - `public_key` - (Required) The public SSH key to be added.
-- `organization_id` - (Defaults to [provider](../index.md#organization_id) `organization_id`) The ID of the organization the SSH key is associated with.
 - `project_id` - (Defaults to [provider](../index.md#project_id) `project_id`) The ID of the project the SSH key is associated with.
 
 ## Attributes Reference
 
 In addition to all above arguments, the following attributes are exported:
 
 - `id` - The ID of the SSH key.
+- `organization_id` - The ID of the organization the SSH key is associated with.
```
Comment (kindermoumoute) [inline docs/resources/account_ssh_key.md:33]:
```suggestion
- `organization_id` - The organization ID the SSH key is associated with.
```

File: docs/resources/baremetal_server.md [hunk 4]
```diff
@@ -71,6 +60,7 @@ In addition to all above arguments, the following attributes are exported:
     - `reverse` - The reverse of the IP.
     - `type` - The type of the IP.
 - `domain` - The domain of the server.
+- `organization_id` - The ID of the organization the server is associated with.
```
Comment (kindermoumoute) [inline docs/resources/baremetal_server.md:63]:
```suggestion
- `organization_id` - The organization ID the server is associated with.
```

File: docs/resources/instance_ip.md [hunk 5]
```diff
@@ -29,6 +28,7 @@ In addition to all above arguments, the following attributes are exported:
 - `id` - The ID of the IP.
 - `address` - The IP address.
 - `reverse` - The reverse dns attached to this IP
+- `organization_id` - The ID of the organization the IP is associated with.
```
Comment (kindermoumoute) [inline docs/resources/instance_ip.md:31]:
```suggestion
- `organization_id` - The organization ID the IP is associated with.
```

File: docs/resources/instance_placement_group.md [hunk 6]
```diff
@@ -31,6 +30,7 @@ In addition to all above arguments, the following attributes are exported:
 
 - `id` - The ID of the placement group.
 - `policy_respected` - Is true when the policy is respected.
+- `organization_id` - The ID of the organization the placement group is associated with.
```
Comment (kindermoumoute) [inline docs/resources/instance_placement_group.md:33]:
```suggestion
- `organization_id` - The organization ID the placement group is associated with.
```

File: docs/resources/instance_security_group.md [hunk 7]
```diff
@@ -144,6 +143,7 @@ The `inbound_rule` and `outbound_rule` block supports:
 In addition to all above arguments, the following attributes are exported:
 
 - `id` - The ID of the security group.
+- `organization_id` - The ID of the organization the security group is associated with.
```
Comment (kindermoumoute) [inline docs/resources/instance_security_group.md:146]:
```suggestion
- `organization_id` - The organization ID the security group is associated with.
```

File: docs/resources/instance_server.md [hunk 8]
```diff
@@ -203,6 +200,8 @@ In addition to all above arguments, the following attributes are exported:
 - `ipv6_gateway` - The ipv6 gateway address. ( Only set when enable_ipv6 is set to true )
 - `ipv6_prefix_length` - The prefix length of the ipv6 subnet routed to the server. ( Only set when enable_ipv6 is set to true )
 - `boot_type` - The boot Type of the server. Possible values are: `local`, `bootscript` or `rescue`.
+- `organization_id` - The ID of the organization the server is associated with.
```
Comment (kindermoumoute) [inline docs/resources/instance_server.md:203]:
```suggestion
- `organization_id` - The organization ID the server is associated with.
```

File: docs/resources/instance_server.md [hunk 9]
```diff
@@ -203,6 +200,8 @@ In addition to all above arguments, the following attributes are exported:
...
+  If you intend to deploy your instance in another project than the default one use your `project_id` instead of the organization id.
```
Comment (kindermoumoute) [inline docs/resources/instance_server.md:204]: this line is not required anymore

File: docs/resources/instance_volume.md [hunk 10]
```diff
@@ -37,6 +37,7 @@ In addition to all above arguments, the following attributes are exported:
 
 - `id` - The ID of the volume.
 - `server_id` - The id of the associated server.
+- `organization_id` - The ID of the organization the volume is associated with.
```
Comment (kindermoumoute) [inline docs/resources/instance_volume.md:40]:
```suggestion
- `organization_id` - The organization ID the volume is associated with.
```

File: docs/resources/k8s_cluster.md [hunk 11]
```diff
@@ -178,6 +177,7 @@ In addition to all above arguments, the following attributes are exported:
     - `token` - The token to connect to the Kubernetes API server.
 - `status` - The status of the Kubernetes cluster.
 - `upgrade_available` - Set to `true` if a newer Kubernetes version is available.
+- `organization_id` - The ID of the organization the cluster is associated with.
```
Comment (kindermoumoute) [inline docs/resources/k8s_cluster.md:180]:
```suggestion
- `organization_id` - The organization ID the cluster is associated with.
```

File: docs/resources/lb.md [hunk 12]
```diff
@@ -53,6 +51,7 @@ In addition to all arguments above, the following attributes are exported:
 
 - `id` - The ID of the load-balancer.
 - `ip_address` -  The load-balance public IP Address
+- `organization_id` - The ID of the organization the load-balancer is associated with.
```
Comment (kindermoumoute) [inline docs/resources/lb.md:54]:
```suggestion
- `organization_id` - The organization ID the load-balancer is associated with.
```

File: docs/resources/rdb_instance.md [hunk 13]
```diff
@@ -73,7 +69,7 @@ In addition to all arguments above, the following attributes are exported:
     - `port` - Port of the replica.
     - `name` - Name of the replica.
 - `certificate` - Certificate of the database instance.
-
+- `organization_id` - The ID of the organization the Database Instance is associated with.
```
Comment (kindermoumoute) [inline docs/resources/rdb_instance.md:72]:
```suggestion
- `organization_id` - The organization ID the Database Instance is associated with.
```

File: docs/resources/registry_namespace.md [hunk 14]
```diff
@@ -45,6 +43,7 @@ In addition to all arguments above, the following attributes are exported:
 
 - `id` - The ID of the namespace
 - `endpoint` - Endpoint reachable by Docker.
+- `organization_id` - The ID of the organization the namespace is associated with.
```
Comment (kindermoumoute) [inline docs/resources/registry_namespace.md:46]:
```suggestion
- `organization_id` - The organization ID the namespace is associated with.
```

File: docs/resources/vpc_private_network.md [hunk 15]
```diff
@@ -32,7 +32,7 @@ The following arguments are supported:
 In addition to all above arguments, the following attributes are exported:
 
 - `id` - The ID of the private network.
-- `organization_id` - (Defaults to [provider](../index.md#organization_id) `organization_id`) The ID of the organization the private network is associated with.
+- `organization_id` - The ID of the organization the private network is associated with.
```
Comment (kindermoumoute) [inline docs/resources/vpc_private_network.md:35]:
```suggestion
- `organization_id` - The organization ID the private network is associated with.
```

**Review by kindermoumoute — CHANGES_REQUESTED** [review]

File: docs/index.md [hunk 16]
```diff
@@ -103,11 +103,11 @@ resource "scaleway_instance_server" "web" {
 
 ## Authentication
```
Comment (QuentinBrosse) [inline docs/index.md:104]: Maybe we should also remove the `SCW_DEFAULT_ORGANIZATION_ID` below?

**Review by kindermoumoute — APPROVED** [review]
LGTM

**Pull request merged**